src/XMLSecurityKey.php

```diff
@@ -375,7 +368,7 @@ private function encryptMcrypt($data)
         if ($this->cryptParams['mode'] == MCRYPT_MODE_CBC) {
             $bs = mcrypt_enc_get_block_size($td);
             for ($datalen0 = $datalen = strlen($data); (($datalen % $bs) != ($bs - 1)); $datalen++)
```

Comment:
This random padding doesn't have any secure significance, leaving it untouched using `mt_rand`.